scipy/_lib/tests/test_public_api.py

```diff
@@ -234,7 +234,7 @@ def ignore_errors(name):
         # `array_api_compat`. This should only raise if SciPy is configured with
         # that library as an available backend.
         backends = {'cupy': 'cupy',
-                    'pytorch': 'torch',
+                    'torch': 'torch',
```

Comment:
This makes the whole `backends` dict redundant, ISTM

Reply:
true!